tasks/internal/request-and-upload-signature/request-and-upload-signature.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   params:
     - description: A docker image of operator-pipeline-images for the steps to run in
       name: pipeline_image
-      default: "quay.io/konflux-ci/release-service-utils:7312e2ecbe67e973edd1f0031acb490d6c961a41"
+      default: "quay.io/konflux-ci/release-service-utils:34dd4b265f02b0890efdbc7b1be4d1187d8881f0"
```

Comment:
Please link the related utils PR in the PR description and also update all the other references in this folder (tests).